Tests/Helper/Mocks/MockKeyboardAccessoryDelegate.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  MockKeyboardAccessoryDelegate.swift
+//  KeyboardSupport-iOSTests
+//
+//  Created by Earl Gaspard on 1/4/19.
+//  Copyright © 2019 Bottle Rocket. All rights reserved.
+//
+
+import Foundation
+import KeyboardSupport
+
+enum TapType {
+    case back
+    case next
+    case done
+    case none
```

Comment:
Instead of having a separate `none` case for a scenario which isn't really valid, I think making the `tapType` property on `MockKeyboardAccessoryDelegate` optional would better represent the situation where the delegate hasn't received a tap yet.
```swift
class MockKeyboardAccessoryDelegate: KeyboardAccessoryDelegate {
    var tapType: TapType?

    // ...
}
```

Reply:
Sounds good. I'll change it.